bundler/intermediates/CADisigI1CertificationService_cb619f69c2d820b4bc23e4d4339f06a96d6373d4_021c8d95a0308df09c0000000000611515.crt

```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Please remove the new /intermediates from this commit, they were added by accident.